src/core/jbuild

```diff
@@ -1,4 +1,18 @@
 
+(rule
+  ((targets (CCArray.mli))
+   (action (run ${ROOT}/src/unlabel.exe ${<} ${@}))
+   (deps (CCArrayLabels.mli ${ROOT}/src/unlabel.exe))
+   (mode promote)
```

Comment:
"promote" will copy the generated file back to the source tree, correct?

Reply:
correct. They won't be removed during clean, but automatically updated on OCaml >= 4.03.0.